src/modules/handle-curators.ts

```diff
@@ -0,0 +1,36 @@
+import BigNumber from "bignumber.js";
+import db from "src/db";
+import { curators } from "src/db/models/curators";
+
+export async function handleCurators(
+  address: string,
+  totalVotes: BigNumber,
+  councilAmount: number | string,
+  networkId: number
+) {
+  const isCurator = totalVotes.gte(councilAmount);
+  const curatorInDb = await db.curators.findOne({ where: { address, networkId } });
+
+  if (curatorInDb) {
+    curatorInDb.isCurrentlyCurator = isCurator;
+    curatorInDb.tokensLocked = totalVotes.toFixed();
```

Comment:
why are we removing decimal places of the tokens locked? we should keep the value as real to the chain value as possible